workflows/cloudify_system_workflows/snapshots/snapshot_create.py

```diff
@@ -173,24 +172,3 @@ def _get_and_create_snapshots_dir(self):
         if not os.path.exists(snapshots_dir):
             os.makedirs(snapshots_dir)
         return snapshots_dir
-
-    def _save_hash_salt(self):
```

Comment:
I think removing this will mean that users who are restored from managers with different hash_salt values in rest_security.conf will no longer be usable.

Reply:
I'm not sure I understand, in the new snapshot I don't save the hash_salt separately but I save the whole rest-security.conf and then the previous hash_salt will be restored.

Reply:
Oh, whoops- yes, good point, for some reason my brain interpreted this method as saving the actual users' password hashes. It doesn't. Sorry, this is fine!